cookbooks/tile/templates/default/export.erb

```diff
@@ -54,6 +55,115 @@ def output_error(message, status = "400 Bad Request"):
   print("</body>")
   print("</html>")
 
+# Add a copyright notice for raster formats (PNG, JPEG, WEBP)
+def add_copyright_notice_raster(image, map_width, map_height, format):
+  #  Create a temporary file for the PNG image
+  with tempfile.NamedTemporaryFile(suffix=".png") as png_tempfile:
+    # Save the Mapnik image into the temporary file as PNG
+    png = image.tostring("png")
+    png_tempfile.write(png)
```

Comment:
Can we not just do `io.BytesIO(png)` to create an in-memory file object that can be passed to the surface constructor without involving a temporary file?

Reply:
I’m switched to BytesIO instead of temporary files now. I did a short smoke test with png and webp output, and that looked ok. I hope I didn’t miss anything…